istio-telemetry/mixer-telemetry/values.yaml

```diff
@@ -30,6 +30,12 @@ mixer:
       contextGraph:
         enabled: false
 
+      logging:
```

Comment:
Could you make them enable by default? So that the default config does not change.

Reply:
done!